website/blog/2019-05-28-hola.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Should these files be here?

Reply:
@lholmquist I probably should have asked earlier but does using docusaurus for this fit into our Nodedeshift website ?

Reply:
Those are for testing. I see now that nodeshift website can be used for hosting thouse so we might use what is used there.